dashboard/app/models/data_doc.rb

```diff
@@ -24,14 +24,6 @@ def to_param
     key
   end
 
-  def summarize_for_edit
-    {
-      key: key,
-      name: name,
-      content: content
-    }
-  end
```

Comment:
@TurnerRiley I removed this because it was the same method as serialize –– let me know if you think something different would be good

Reply:
Yes looks great!

Reply:
makes sense to me! just noting that end-user facing pages will have translations on them and need separate logic, but serializing to .json file and rendering edit page seem like they have a good chance of needing the same, untranslated data as each other in the future.

Reply:
Not sure I totally understand this –– will ask in our 1:1